src/core/nginx.go

```diff
@@ -284,6 +284,10 @@ func (n *NginxBinaryType) validateConfigCheckResponse(response *bytes.Buffer, co
 		return fmt.Errorf("error running nginx -t -c %v:\n%s", configLocation, response)
```

Comment:
maybe instead of 2 if statements combine them? ```
if (bytes.Contains(response.Bytes(), []byte("[emerg]") || bytes.Contains(response.Bytes(), []byte("[alert]"))) {
		return fmt.Errorf("error running nginx -t -c %v:\n%s", configLocation, response)
}```